website/repairnator-mongo-rest-api/server/controllers/inspector.controller.js

```diff
@@ -103,6 +105,28 @@ function countSuccessFullyReproducedBuild(req, res, next) {
     .catch(e => next(e));
 }
 
+function speedrate(req, res, next) {
+  Inspector.speedrate()
+    .then((values) => {
+      const gtDateIso = moment().startOf('hour').subtract(24, 'hours');
+      // Array with count of 0
+      const hours = _.map(_.range(24), (plusHour) => {
```

Comment:
Might just be my inexperience with javascript, but what is plushour here? I assume it has something to do with the `subtract(24, 'hours');` above?

Reply:
It is the argument, the number of hours we add to gtDateIso, now minus 24 hours. 
The goal is to create an array of 24 dates corresponding to the 24 past hours.

Reply:
Okay.